client/client.go

```diff
@@ -257,6 +257,12 @@ func (c *Client) processChannelRequests() {
 
 		case *agentpb.StartActionRequest:
 			var action actions.Action
+			actionTimeout := p.GetTimeout()
```

Comment:
Maybe it's better to place here https://github.com/percona/pmm-agent/pull/180/files#diff-1cd41a2611da49ce63630d5a51bd82c5R314? Because we define variable action, then set actionTimeout and then come back to action